packages/platform-browser/src/security/dom_sanitization_service.ts

```diff
@@ -173,7 +176,9 @@ export class DomSanitizerImpl extends DomSanitizer {
         if (allowSanitizationBypassOrThrow(value, BypassType.Script)) {
           return unwrapSafeValue(value);
         }
-        throw new Error('unsafe value used in a script context');
+        throw new RuntimeError(
+            RuntimeErrorCode.SANITIZATION_UNSAFE_SCRIPT,
+            ngDevMode && 'unsafe value used in a script context');
```

Comment:
We should probably use `NG_DEV_MODE` here as well (and declare a const above in this file):
```suggestion
            NG_DEV_MODE && 'unsafe value used in a script context');
```

Reply:
I've seen both used in the code base, what is the purpose of that double check with `NG_DEV_MODE` ?

Reply:
This is a bit inconsistent in the code at this moment (we plan to improve that in the future). There are places of code where `ngDevMode` is guaranteed to be defined (most of the runtime code in core), so we use it without extra checks. In other cases, we check whether the `ngDevMode` is defined first. The unification that we are thinking of (not yet finalized/decided) is to switch to importing the `ngDevMode` from a file (vs having is as a global), but it would have effect on generated code as well, so this is a larger design exploration that we'd need to do in the future.